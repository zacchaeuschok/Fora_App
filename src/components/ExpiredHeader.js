import React from "react";
import { View, Text, Image, TextInput, StatusBar } from "react-native";
import { useNavigation, useIsFocused } from "@react-navigation/native";
import { CircleButton } from "./Button";
import { supabase } from "../initSupabase";
import { useState, useEffect } from 'react'

import { COLORS, FONTS, SIZES, assets } from "../constants";
//import from from "../../__mocks__/@react-native-async-storage/async-storage";

export const ExpiredHeader = () => {
  const navigation = useNavigation();

  return (
    <View
      style={{
        flex: 3,
        paddingHorizontal: 20,
        paddingBottom: 20,
      }}
    >
      <CircleButton
        imgUrl={assets.left}
        handlePress={() => navigation.goBack()}
        left={15}
        top={StatusBar.currentHeight + 10}
      />
      <Text
        style={{
          fontWeight: "bold",
          alignSelf: "center",
          padding: 30,
          color:COLORS.white,
          fontSize: SIZES.large
        }}
      >
        Expired Question
      </Text>
    </View>
  );
};
