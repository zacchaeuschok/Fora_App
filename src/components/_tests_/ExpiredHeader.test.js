import { ExpiredHeader } from '../ExpiredHeader'
import renderer from 'react-test-renderer'
import Navigation from "../../navigation"
import { fireEvent, render } from '@testing-library/react-native'

jest.useFakeTimers()

it('renders correctly', () => {
    const tree = renderer.create(
      <Navigation>
        <ExpiredHeader/>
      </Navigation>
    )
    expect(tree).toMatchSnapshot()
  })

  