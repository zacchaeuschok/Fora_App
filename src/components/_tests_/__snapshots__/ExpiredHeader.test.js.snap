// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RNCSafeAreaView
  style={
    Array [
      undefined,
      Object {
        "backgroundColor": "#f7f7f7",
        "flex": 1,
      },
    ]
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <ActivityIndicator
      animating={true}
      color="#3366FF"
      hidesWhenStopped={true}
      size="large"
    />
  </View>
</RNCSafeAreaView>
`;
